test-bot/src/main/kotlin/com/kotlindiscord/kord/extensions/testbot/extensions/I18nTestExtension.kt

```diff
@@ -94,6 +95,23 @@ public class I18nTestExtension : Extension() {
                 }
             }
         }
+
+        publicSlashCommand(::I18nTestNamedArguments) {
+            name = "command.apple"
+            description = "command.apple"
+
+            action {
+                respond {
+                    content = translate(
+                        "command.apple.response",
+                        mapOf(
```

Comment:
This needs a newline before it.

Reply:
Same here, it didn't update in here for some reason, But if you check the commit, it is done.